code-challenges.test.js

```diff
@@ -15,35 +15,138 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+// input: colors1
+// output: Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// input: colors2
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+describe("shiftSortedArray", () => {
+  const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+  const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
```

Comment:
Your tests look great!  Consider placing your test variables inside the it block just before the expect method as this is the scope in which they are being used.